### PR TITLE
Discuss security considerations of racket/sandbox in Reference

### PR DESCRIPTION
**Pull request by thinkmoore**

### CODE REVIEW

File: pkgs/racket-doc/scribblings/reference/sandbox.scrbl [hunk 1]
```diff
@@ -300,6 +286,30 @@ is terminated.  Once a sandbox raises such an exception, it will
...
+exercise arbitrary functionality from the Racket runtime and installed collections,
+an attacker who identifies a vulnerability in Racket or an installed collection
+may be able to escape the sandbox.
+
+To mitigate this risk, programs that use the sandbox should employ additional
+precautions when possible. Suggested measures include:
+@itemlist[
+@item{Supplying a custom module language to @racket[make-evaluator] or
+@racket[make-module-evaluator] that gives untrusted code access to only
+the language constructs it absolutely requires.}
```
Comment (samth) [inline pkgs/racket-doc/scribblings/reference/sandbox.scrbl]: This might mention the handin server as an example.

**Comment by mflatt** [comment]
This looks good overall to me.

**Comment by thinkmoore** [comment]
I don't mind either way about adding a mention of the handin server. This ought to get merged for 6.9 though.

**Comment by stamourv** [comment]
Merged, thanks! And will merge to 6.9.
Discussion of the handin server can be added later.

**Pull request closed without merging**